<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SetCamerasPrtWidget</class>
 <widget class="QWidget" name="SetCamerasPrtWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1383</width>
    <height>870</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(63,63,70);
color: rgb(255, 255, 255);</string>
  </property>
  <widget class="QWidget" name="gridLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>1371</width>
     <height>851</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <widget class="QGroupBox" name="groupBox">
      <property name="title">
       <string>相机1-1</string>
      </property>
      <widget class="CammerWidget" name="Cam1_1" native="true">
       <property name="geometry">
        <rect>
         <x>9</x>
         <y>19</y>
         <width>320</width>
         <height>390</height>
        </rect>
       </property>
      </widget>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QGroupBox" name="groupBox_2">
      <property name="title">
       <string>相机1-2</string>
      </property>
      <widget class="CammerWidget" name="Cam1_2" native="true">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>320</width>
         <height>390</height>
        </rect>
       </property>
      </widget>
     </widget>
    </item>
    <item row="0" column="3">
     <widget class="QGroupBox" name="groupBox_4">
      <property name="title">
       <string>相机2-2</string>
      </property>
      <widget class="CammerWidget" name="Cam2_2" native="true">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>320</width>
         <height>390</height>
        </rect>
       </property>
      </widget>
     </widget>
    </item>
    <item row="0" column="2">
     <widget class="QGroupBox" name="groupBox_3">
      <property name="title">
       <string>相机2-1</string>
      </property>
      <widget class="CammerWidget" name="Cam2_1" native="true">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>320</width>
         <height>390</height>
        </rect>
       </property>
      </widget>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QGroupBox" name="groupBox_5">
      <property name="title">
       <string>相机3-1</string>
      </property>
      <widget class="CammerWidget" name="Cam3_1" native="true">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>320</width>
         <height>390</height>
        </rect>
       </property>
      </widget>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QGroupBox" name="groupBox_6">
      <property name="title">
       <string>相机3-2</string>
      </property>
      <widget class="CammerWidget" name="Cam3_2" native="true">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>320</width>
         <height>390</height>
        </rect>
       </property>
      </widget>
     </widget>
    </item>
    <item row="1" column="2">
     <widget class="QGroupBox" name="groupBox_7">
      <property name="title">
       <string>相机4-1</string>
      </property>
      <widget class="CammerWidget" name="Cam4_1" native="true">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>320</width>
         <height>390</height>
        </rect>
       </property>
      </widget>
     </widget>
    </item>
    <item row="1" column="3">
     <widget class="QGroupBox" name="groupBox_8">
      <property name="title">
       <string>相机4-2</string>
      </property>
      <widget class="CammerWidget" name="Cam4_2" native="true">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>320</width>
         <height>390</height>
        </rect>
       </property>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CammerWidget</class>
   <extends>QWidget</extends>
   <header location="global">Src/cammerwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
